File: 2.Implement/Delivery_Server/src/main/java/com/delivery/app/Delivery/data/dto/response/review/SaveReviewResponseDto.java
package com.delivery.app.Delivery.data.dto.response.review;

import com.delivery.app.Delivery.data.dto.response.BaseResponseDto;
import lombok.Getter;

@Getter
public class SaveReviewResponseDto implements BaseResponseDto {
    private Long id;

    public SaveReviewResponseDto(Long id) {
        this.id = id;
    }
}

File: 2.Implement/DeliveryApp/app/src/main/java/com/project/deliveryapp/retrofit/dto/response/review/SaveReviewResponseDto.kt
package com.project.deliveryapp.retrofit.dto.response.review

import com.project.deliveryapp.retrofit.dto.ResponseDto
import com.project.deliveryapp.toList
import org.json.JSONObject
import java.time.LocalDateTime

class SaveReviewResponseDto(json: JSONObject): ResponseDto {
    val id: Long

    init {
        val obj = json.getJSONObject("responseDto")

        id = obj.getLong("id")
    }

}